tests/components/samsungtv/test_media_player.py

```diff
@@ -334,36 +390,30 @@ def mock_now():
     assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_TV
 
 
-async def test_turn_off_websocket(hass, remote):
-    """Test for turn_off."""
-    await setup_samsungtv(hass, MOCK_CONFIG)
-    assert await hass.services.async_call(
-        DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: ENTITY_ID}, True
-    )
-    # key called
-    assert remote.control.call_count == 1
-    assert remote.control.call_args_list == [call("KEY_POWER")]
-
-
-async def test_turn_off_legacy(hass):
+async def test_turn_off_websocket(hass, remotews):
     """Test for turn_off."""
-    with patch("homeassistant.components.samsungtv.config_flow.socket"), patch(
-        "homeassistant.components.samsungtv.config_flow.Remote",
+    with patch(
+        "homeassistant.components.samsungtv.bridge.Remote",
         side_effect=[OSError("Boom"), mock.DEFAULT],
-    ), patch(
-        "homeassistant.components.samsungtv.media_player.SamsungRemote"
-    ) as remote_class, patch(
-        "homeassistant.components.samsungtv.socket"
     ):
-        remote = mock.Mock()
-        remote_class.return_value = remote
-        await setup_samsungtv(hass, MOCK_CONFIG_NOTURNON)
+        await setup_samsungtv(hass, MOCK_CONFIGWS)
         assert await hass.services.async_call(
-            DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: ENTITY_ID_NOTURNON}, True
+            DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: ENTITY_ID}, True
         )
         # key called
-        assert remote.control.call_count == 1
-        assert remote.control.call_args_list == [call("KEY_POWEROFF")]
+        assert remotews.send_key.call_count == 1
+        assert remotews.send_key.call_args_list == [call("KEY_POWER")]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5152307)